src/txmempool.cpp

```diff
@@ -406,7 +406,7 @@ void CTxMemPool::removeConflicts(const CTransaction &tx, std::list<CTransaction>
     }
 }
 
-void CTxMemPool::removeExpired(unsigned int nBlockHeight)
+std::vector<std::string> CTxMemPool::removeExpired(unsigned int nBlockHeight)
```

Comment:
Hmm.. thinking this over again, we should instead be returning:
```suggestion
std::vector<uint256> CTxMemPool::removeExpired(unsigned int nBlockHeight)
```